### ISSUE TITLE
Better version numbering system

### ISSUE THREAD

**Issue by Tyler-Ward** [srobo/ops-manual#3]
The current version numbering system is likley to end up in some quite large numbers being produced as small edits/typos are made Sugest moving to a style similar to https://semver.org/

e.g. 3.1.2
Main version number: Major changes/new structures ect
Minor version number: Small changes such as small alterations to procedures
Maintanace version number: Clarifications and typos.

**Comment by Tyler-Ward** [comment]
@richardbarlow give you have just released v5 i assume you haven't spotted this.

**Comment by richardbarlow** [comment]
@Tyler-Ward I've just added a page on 'Release Versioning' that specified that the ops manual will follow semantic versioning: https://github.com/srobo/ops-manual/commit/bd71721fdf321b70f391bb41f2769389e6b52e8b. On that page I've tried to explain how it will be applied to the ops manual. I've also added a note to the Change Log page to state that it adheres to Keep a Changelog v1.0.0 (https://keepachangelog.com/en/1.0.0/): https://github.com/srobo/ops-manual/commit/9796fc67946c7438ee9e9fbbcadfcc4b1fe68499.
